rules.ini

```diff
@@ -114,3 +114,4 @@ Steam_Audio = (?:^|/)(?:lib)?steamaudio\.(?:dylib|dll|so)$
 Steam_Networking = (?:^|/)(?:lib)?steamnetworkingsockets\.(?:dylib|dll|so)$
 Tobii = (?:^|/)Tobii
 NodeJS = (?:^|/)node\.dll$
+Qt = (?:^|/)Qt(?:\d|)Core(?:\d|)\.dll$
```

Comment:
Let's not get ahead of our selves and match what we know.

`(?:^|/)Qt(?:Core4|5Core)\.dll$` should to the trick.

Reply:
Done.

Reply:
And yeah, it's `Qt6Core.dll`, I'll pick up the second pattern.